Model/Request/Buyers.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+namespace Balancepay\Balancepay\Model\Request;
+
+use Balancepay\Balancepay\Model\AbstractRequest;
+use Balancepay\Balancepay\Model\Response\Factory as ResponseFactory;
+use Magento\Customer\Api\AccountManagementInterface;
+use Magento\Directory\Model\RegionFactory;
+use Magento\Framework\App\RequestInterface;
+use Balancepay\Balancepay\Model\Config;
+use Balancepay\Balancepay\Lib\Http\Client\Curl;
+use Balancepay\Balancepay\Helper\Data as HelperData;
+use Magento\Customer\Model\Session;
+use Magento\Customer\Model\Address;
+
+/**
+ * Balancepay webhooks request model.
+ */
+class Buyers extends AbstractRequest
```

Comment:
Maybe rename to `BuyersTopicWebhookRequest`?

Reply:
This is not related to only webhook request, we also call differenct buyer related APIs using this